cds-snc/cppd-medical-report
# Removed 'level of'

# Description

No Trello card. Sort of a test first pull request – Removed 'level' from the question about dosage, as it's a tautology. Also shortened 'February' to 'Feb' for the symptoms starting hint text for consistency.

# Test Instructions

Go to the conditions page
Look at the 'When symptoms began' question hint text. It should say 'Feb 2008'.

Go to the medications page.
The second question should be 'What dosage does the patient take?'

# Help Requested

Please check that the text displays as described in the test. Non-devs are fine to test this.
